DXMainClient/DXMainClient.csproj

```diff
@@ -6,16 +6,7 @@
     <Prefer32Bit Condition="'$(Engine)' == 'WindowsXNA' And '$(TargetFrameworkIdentifier)' == '.NETFramework'">true</Prefer32Bit>
     <UseAppHost>false</UseAppHost>
     <SelfContained>false</SelfContained>
-    <Title>CnCNet Client</Title>
     <Description>CnCNet Main Client Library</Description>
-    <Company>CnCNet</Company>
-    <Product>CnCNet Client</Product>
-    <Copyright>Copyright © CnCNet, Rampastring 2011-2024</Copyright>
-    <Trademark>CnCNet</Trademark>
-    <Version>2.8.0.0</Version>
```

Comment:
Does this mean the version will be managed by gitversion?

Reply:
> Does this mean the version will be managed by gitversion?

yes